gef-ui/templates/includes/application-details/about-exporter.njk

```diff
@@ -19,7 +19,7 @@
   </div>
 </div>
 
-{% if exporter.status.code === 'NOT_STARTED' and not previewMode %}
+{% if exporter.status.code === 'Not started' and not previewMode %}
```

Comment:
Can we centralised constants on njk as well to keep it consistent across all the templates, any thoughts?
Could be a future PR just a thought.

Reply:
future PR